package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "private": true,
   "engines": {
-    "node": "8.x.x",
+    "node": "8.12.x",
```

Comment:
We can talk in the channel, but we should coordinating around  getting our apps to Node 10. cc @zephraph

Reply:
Yeah, we should start that convo